controllers/openshift_controller.go

```diff
@@ -153,9 +153,8 @@ func (r *KataConfigOpenShiftReconciler) processDaemonsetForCR(operation DaemonOp
 					NodeSelector:       nodeSelector,
 					Containers: []corev1.Container{
 						{
-							Name: "kata-install-pod",
-							// Image:           "quay.io/isolatedcontainers/kata-operator-daemon:v1.0",
-							Image:           "docker.io/pharshal/kata-install-daemon:0.1.1",
+							Name:            "kata-install-pod",
```

Comment:
In different places, I found `kata-install-daemon`, `kata-operator-daemon`, `kata-install-pod`. This can be confusing a bit, I think consistency here is relevant.

Also, here it is called `kata-install-pod` but its really just a container in a pod in that context.

Reply:
Yes, I'll fix the README. I was about to do it in pr #21 but closed it for some reason. Anyway thanks for the hint!
And about the different namings, I also agree and will open an issue for it.